change/@fluentui-react-theme-29a85799-2552-40b5-bf1b-9c0c4372f1a4.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "react-theme: Exporting token to css variable mapping.",
```

```suggestion
  "comment": "feat: Export tokens as mapping to CSS variables",
```

_nit_ As we have grouped changelog there is no need to specify package name